File: src/main/java/com/dingdo/DingDoRobot.java
package com.dingdo;

import org.mybatis.spring.annotation.MapperScan;
import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

/**
 * 一些声明信息
 *
 * @author slamacraft
 * @date: 2020/8/10 15:12
 * @since JDK 1.8
 */
@MapperScan(basePackages = {"com.dingdo.**.mapper"})
@EnableScheduling
@SpringBootApplication
public class DingDoRobot {

    public static void main(String[] args) {
        SpringApplication springApplication = new SpringApplication(DingDoRobot.class);
        springApplication.run(args);
    }
}

File: src/main/scala/com/dingdo/DingDoRobot.scala
package com.dingdo


//import org.mybatis.spring.annotation.MapperScan
import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.scheduling.annotation.EnableScheduling
import springfox.documentation.swagger2.annotations.EnableSwagger2

//@MapperScan(basePackages = Array("com.dingdo.**.mapper"))
@EnableSwagger2
@EnableScheduling
@SpringBootApplication
class DingDoRobot

object DingDoRobot extends App {
  val springApplication = new SpringApplication(classOf[DingDoRobot])
  springApplication.run(args: _*)
}